### PR TITLE
Fix old New Relic plugin.

### PR DESCRIPTION
**Pull request by pjambet**
This reads the current requests per minute from new relic. It can be
used with google analytics, and dynosaur will use the max estimate of
both plugins.

The spec I added is pretty minimalistic, I'll see if I can add more.

### CODE REVIEW

File: lib/dynosaur/inputs/new_relic_input_plugin.rb [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        @new_relic_api_client = Dynosaur::NewRelicApiClient.new(@key, @app_id)
+      end
+
+      def retrieve
+        return get_rpm
+      end
+
+      def value_to_resources(value)
+        estimate = (value / @rpm_per_dyno.to_f).ceil
+        puts "NEW RELIC: #{@recent} => #{value} : #{estimate}"
```
Comment (pjambet) [inline lib/dynosaur/inputs/new_relic_input_plugin.rb]: @werkshy Should we log this? This was in the previous version of the plugin, but it doesn't seem like we do it in the GA plugin.

File: lib/dynosaur/inputs/new_relic_input_plugin.rb [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+      private
+
+      def get_rpm
+        begin
+          # New Relic is not very accurate and sometimes returns data for the
+          # current minute, as if it was already complete.
+          # This is not an issue here because we always use the max value from
+          # the ring buffer. This would become an issue if we had an
+          # hysteresis_period smaller than 1 minute.
```
Comment (pjambet) [inline lib/dynosaur/inputs/new_relic_input_plugin.rb]: we might want to raise an exception, or maybe issue a warning if the hysteresis_period is shorter than a minute

File: lib/dynosaur/inputs/new_relic_input_plugin.rb [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    class NewRelicInputPlugin < AbstractInputPlugin
+
+      # Load config from the config hash
+      def initialize(config)
+        super
+
+        @unit = "RPM"
+        @key = config["key"]
+        @app_id = config["app_id"].to_s
+        @rpm_per_dyno = config["rpm_per_dyno"].to_i
```
Comment (werkshy) [inline lib/dynosaur/inputs/new_relic_input_plugin.rb]: If this is blank should we have a default?

File: lib/dynosaur/inputs/new_relic_input_plugin.rb (hunk 2 above)
Comment (werkshy) [inline lib/dynosaur/inputs/new_relic_input_plugin.rb]: Agreed

**Comment by werkshy** [comment]
Minor comments only: fix and merge at your will.

File: lib/dynosaur/inputs/new_relic_input_plugin.rb (hunk 2 above)
Comment (pjambet) [inline lib/dynosaur/inputs/new_relic_input_plugin.rb]: exception or warning? :)

File: lib/dynosaur/inputs/new_relic_input_plugin.rb (hunk 3 above)
Comment (pjambet) [inline lib/dynosaur/inputs/new_relic_input_plugin.rb]: Yeah, that'd be good, it seems like ga doesn't have a default either, want me to add one?

Comment (werkshy) [inline lib/dynosaur/inputs/new_relic_input_plugin.rb]: Sure.

**Pull request merged**